draft-ietf-quic-recovery.md

```diff
@@ -59,12 +44,7 @@ normative:
 
 informative:
 
-  RFC3782:
```

Comment:
In which case you're supposed to mention them somewhere in the text (otherwise idnits will give you warnings).